docs/learn/expert-guides/set-token-metadata-on-deploy.md

```diff
@@ -0,0 +1,191 @@
+---
+sidebar_label: '💽 Set Token Metadata on deployment'
```

Comment:
How about just calling it

```suggestion
sidebar_label: '💽 Deploy Token with Metadata'
```

@Hugoo @fhildeb wdyt?

Reply:
Good idea. Will apply to file and heading